### PR TITLE
Expand automated testing with data collection to MIDI and audio input

### PR DESCRIPTION
**Pull request by BenjaminFair**

### CODE REVIEW

File: android/WALT/app/src/main/java/org/chromium/latency/walt/AudioTest.java [hunk 1]
```diff
@@ -279,4 +311,18 @@ private void finishAndShowStats() {
             resultHandler.onResult(deltas);
         }
     }
+
+    private void finishRecordingTest() {
+        clockManager.checkDrift();
+
+        logger.log("deltas: " + deltas.toString());
+        logger.log(String.format(Locale.US,
+                "Median audio latency %.1f ms",
```
Comment (kamrik) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/AudioTest.java]: Let's add "recording" to the phrasing here, to make it easy to distinguish from the input when reading the log.

Comment (BenjaminFair) [inline android/WALT/app/src/main/java/org/chromium/latency/walt/AudioTest.java]: Done

**Comment by BenjaminFair** [comment]
PTAL

**Comment by kamrik** [comment]
Looks good, merging.

**Comment by gkasten** [comment]
+1

**Pull request merged**